### ISSUE TITLE
autohide_bookmarks_and_main_toolbars does not show bookmarks bar if window is in full screen (macOS)

### ISSUE THREAD

**Issue by GJRobert** [MrOtherGuy/firefox-csshacks#327]
On macOS,

Window in full-screen mode:
<img width="423" alt="截圖 2023-11-23 16 32 57" src="https://github.com/MrOtherGuy/firefox-csshacks/assets/655584/4914f087-3da9-4b83-a162-3f02031c96a4">

Not in full-screen mode:
<img width="427" alt="截圖 2023-11-23 16 33 09" src="https://github.com/MrOtherGuy/firefox-csshacks/assets/655584/058f8c29-6208-4db3-99cd-15b8970149b4">

**Comment by MrOtherGuy** [comment]
Indeed, that's intentional. Or rather, we are not changing the normal behavior of Firefox in that regard - if you disable all custom css rules then you should see that Firefox does not show bookmarks toolbar when in fullscreen mode.

**Comment by MrOtherGuy** [comment]
However I see that this style was kinda broken in fullscreen mode in another manner - the content area was pushed too far up if Firefox internal option to hide toolbars in fullscreen mode was selected. This should be fixed in 4f3ac16
